### ISSUE TITLE
AFD Section Parse Errors

### ISSUE THREAD

**Issue by eric-gade** [weather-gov/weather.gov#1830]
### Description
This issue is actually two problems in one:
1. Whenever something unexpected comes back in parsed AFD nodes -- specifically when it comes to the temps/pops logic -- a twig failure breaks the whole site;
2. A specific case (bundled below) of questionably formatted AFD sections breaks twig assumptions

Do this at same time as #1796 

### Additional detail
You can see the formatting problem in this part of the captured AFD:
```
.PRELIMINARY POINT TEMPS/POPS...
TPA  89  79  88  78 /  10  10  50  40 
FMY  90  80  90  79 /  50  60  80  40 
GIF  92  77  91  77 /  20  20  70  30 
SRQ  90  79  89  78 /  20  30  60  50 
BKV  89  74  90  74 /  10  10  40  30 
SPG  90  82  89  81 /  10  20  50  50 

&&

Sea Breeze Thunderstorm Regime For Friday: 5
Sea Breeze Thunderstorm Regime For Saturday: 5

For additional information on sea breeze regimes, go to: 
     https://www.weather.gov/tbw/ThunderstormClimatology


&&
```

Either our reading of the directive is incorrect, or this is not properly formatted.
### Acceptance criteria

- [ ] Tests have been implemented or modified to capture the bug
- [ ] The example bundle renders correctly
- [ ] Twig render errors for some subsection of the page do not break the whole site (could be kicked to a separate, more general issue if need be)
[bundle_66f6ccf90d614.zip](https://github.com/user-attachments/files/17166290/bundle_66f6ccf90d614.zip)

**Comment by colinmurphy01** [comment]
This reads like a critical bug we should fix this sprint?

CC @greg-does-weather

**Comment by jamestranovich-noaa** [comment]
side note, we have NR logging already setup for the api interop branch so parsing breakage should be easier to track going forward

**Comment by mgwalker** [comment]
I dunno. Since the site isn't operational yet, I think I'd prefer making this fix only in the interop layer. It seems silly to me to fix code that we hope to retire in a couple of weeks.

**Comment by colinmurphy01** [comment]
Sounds like a plan. Will that be included in #1796?

**Comment by greg-does-weather** [comment]
That seems right! I'll note it over there too so we don't forget.

**Comment by colinmurphy01** [comment]
Refinement: 
- Fix the breaking issue with the twig templates
- Update parsing to show text that's not under headers

**Comment by colinmurphy01** [comment]
@eric-gade will check if a temporary fix was implemented?

**Comment by colinmurphy01** [comment]
@eric-gade assigning this to you for this sprint to see if you have this fixed already.

**Comment by greg-does-weather** [comment]
Closed by #2030
